Content.Server/Defusable/Systems/DefusableSystem.cs

```diff
@@ -0,0 +1,228 @@
+    {
+        if (!args.IsInDetailsRange)
+            return;
+
+        if (!comp.BombUsable)
+        {
+            args.PushMarkup(Loc.GetString("defusable-examine-defused", ("name", uid)));
+        }
+        else if (comp.BombLive && TryComp<ActiveTimerTriggerComponent>(uid, out var activeComp))
+        {
+            if (comp.BombDisplayTime)
+            {
+                args.PushMarkup(Loc.GetString("defusable-examine-live", ("name", uid),
+                    ("time", MathF.Floor(activeComp.TimeRemaining))));
+            }
+            else
+            {
+                args.PushMarkup(Loc.GetString("defusable-examine-live-display-off", ("name", uid)));
+            }
+        }
+        else
+        {
+            args.PushMarkup(Loc.GetString("defusable-examine-inactive", ("name", uid)));
+        }
+    }
+
+    #endregion
+
+    #region Anchorable
+
+    private void OnAnchorAttempt(EntityUid uid, DefusableComponent component, AnchorAttemptEvent args)
+    {
+        CheckAnchorAttempt(uid, component, args);
+    }
+
+    private void OnUnanchorAttempt(EntityUid uid, DefusableComponent component, UnanchorAttemptEvent args)
+    {
+        CheckAnchorAttempt(uid, component, args);
+    }
+
+    private void CheckAnchorAttempt(EntityUid uid, DefusableComponent component, BaseAnchoredAttemptEvent args)
+    {
+        // Don't allow the thing to be anchored if bolted
+        if (component.Bolted)
+        {
+            var msg = Loc.GetString("defusable-popup-cant-anchor", ("name", uid));
+            _popup.PopupEntity(msg, uid, args.User);
+
+            args.Cancel();
+        }
+    }
+    #endregion
+
+
+    #region Public
+
+    public void TryDelay(EntityUid uid, float amount, ActiveTimerTriggerComponent? comp = null)
+    {
+        if (!Resolve(uid, ref comp, false))
+            return;
+
+        comp.TimeRemaining += amount;
+    }
+
+    public void TryStartCountdown(EntityUid uid, DefusableComponent comp)
+    {
+        if (!comp.BombUsable)
+        {
+            _popup.PopupEntity(Loc.GetString("defusable-popup-fried", ("name", uid)), uid);
+            return;
+        }
+
+        var xform = Transform(uid);
+        xform.Anchored = true;
+        comp.Bolted = true;
+
+        comp.BombLive = true;
+        _popup.PopupEntity(Loc.GetString("defusable-popup-begun", ("name", uid)), uid);
+        if (TryComp<OnUseTimerTriggerComponent>(uid, out var timerTrigger))
+        {
+            _trigger.HandleTimerTrigger(
+                uid,
+                null,
+                timerTrigger.Delay,
+                timerTrigger.BeepInterval,
+                timerTrigger.InitialBeepDelay,
+                timerTrigger.BeepSound
+            );
+        }
+
+        RaiseLocalEvent(uid, new BombArmedEvent(uid), false);
+
+        UpdateAppearance(uid, comp);
+    }
+
+    public void TryDetonateBomb(EntityUid uid, DefusableComponent comp)
+    {
+        // also might want to have admin logs
```

Comment:
double check that starting/detonating/defusing the bomb has logs, add them if they dont